lib/browser/guest-view-manager.js

```diff
@@ -342,6 +345,17 @@ const handleMessage = function (channel, handler) {
   })
 }
 
+const getGuestForWebContents = function (guestInstanceId, contents) {
```

Comment:
perhaps fold `getGuest` into this function? no particular need to have it separate any more, right?

Reply:
@nornagon original `getGuest` is still when handling `ELECTRON_GUEST_VIEW_MANAGER_FOCUS_CHANGE` as in this case it's the `<webview>` itself sending the IPC and the `guestInstanceId` validation is different.

Reply:
Ah, ok. Well in that case, at least put this function next to `getGuest` :)

Reply:
moved